Fix browsing (#99)
ankitects/anki-manual: src/browsing.md

If you regularly search for the same thing,
you can save the current search by right-clicking the topmost item in the sidebar,
choosing “Save Current Search” and typing in a name.
You can also drag and drop any sidebar item onto this area to add an equivalent

> saved search, basically pinning it to the top.

Reviewer: ```suggestion
saved search, effectively pinning it at the top.
```

This wording may just be personal preference

| Name | Action |
|-|-|
| Change Deck | Move currently selected cards to a different deck. |

> | Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will give new cards an interval between 60 and 90 days, which can be useful when you've imported a bunch of material you already know.

Author: Not sure if I got this right, but I think only new cards receive the interval from Set Due Date, whereas an existing interval won't be modified. Is that correct?

Reviewer: ```suggestion
| Set Due Date | Turn cards into review cards, and make them due on a certain date. This can be useful for moving cards forward or back a few days when your study schedule is interrupted. Entering a range like `60-90` will make the selected cards due between 60 and 90 days from now. New cards will have their interval set to the same delay, but reviews will be rescheduled without changing their current interval, unless '!' is included at the end of the range.
```

Reviewer: Yep, but it might be worth mentioning that reviews are still rescheduled

| Forget | Move currently selected cards to the end of the new queue. The existing review history is preserved. |

> | Reposition | Change the order new cards will appear in. You can find out the existing positions by enabling the _due_ column, as described in the [table](#cardnote-table) section above. If you run the reposition command when multiple cards are selected, it will apply increasing numbers to each card in turn. By default the number increases by one for each card, but this can be adjusted by changing the "step" setting. The **Shift position of existing cards** option allows you to insert cards between currently existing ones, pushing the currently existing ones apart. For instance, if you have five cards and you want to move 3, 4, and 5 between 1 and 2, selecting this setting would cause the cards to end up in the order 1, 3, 4, 5, 2. By contrast, if you turn this option off, 1 and 2 will get the same position number (and it will thus be unpredictable which of the cards with the same number comes up first). Please note that when enabled, any card with a higher position will be modified, and all of those changed cards will need to be sent the next time you sync. |

Author: For your convenience, the change here was removing a space and a pipe at the end.

Reviewer: Thanks :-)

## Find and Replace

This dialog allows for replacing text on notes. As described above, it is available
from the toolbar and the table's context menu.

The first input field is for the text that is going to be replaced, the second
one for the replacement. Next, there is a dropdown menu that allows you to specify
where Anki should look for text to replace: in a note's tags (requires Anki 2.1.45+),
in all fields, or just in a specific field (only fields belonging to a selected
note will be listed).

By default, only selected notes will be affected. If you want to lift that

> restriction, you can untick the first checkbox (requires Anki 2.1.45+).

Reviewer: ```suggestion
restriction, you can untick the "selected notes only" checkbox (requires Anki 2.1.45+).
```

Perhaps naming the checkbox instead of referring to its position is a bit less likely to fall out of sync when changes are made to the interface in the future?
